apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -229,43 +236,63 @@ const $usedProFeatures = computed(
     }
     // pages with dynamic paths
     for (const page of [pages.homePage, ...pages.pages]) {
+      const awareness = {
+        pageId: page.id,
+        instanceSelector: [page.rootInstanceId],
+      };
       if (isPathnamePattern(page.path)) {
-        features.set("Dynamic path", {
-          pageId: page.id,
-          instanceSelector: [page.rootInstanceId],
-        });
+        features.set("Dynamic path", { awareness });
       }
       if (page.meta.status && page.meta.status !== `200`) {
-        features.set("Page status code", {
-          pageId: page.id,
-          instanceSelector: [page.rootInstanceId],
-        });
+        features.set("Page status code", { awareness });
       }
       if (page.meta.redirect && page.meta.redirect !== `""`) {
-        features.set("Redirect", {
-          pageId: page.id,
-          instanceSelector: [page.rootInstanceId],
-        });
+        features.set("Redirect", { awareness });
       }
     }
     // has resource variables
     for (const dataSource of dataSources.values()) {
       if (dataSource.type === "resource") {
         const instanceId = dataSource.scopeInstanceId ?? "";
-        features.set(
-          "Resource variable",
-          findAwarenessByInstanceId(pages, instances, instanceId)
-        );
+        features.set("Resource variable", {
+          awareness: findAwarenessByInstanceId(pages, instances, instanceId),
+        });
       }
     }
-    // instances with animations
+
+    // Instances with animations.
     for (const instance of instances.values()) {
       const [namespace] = parseComponentName(instance.component);
       if (namespace === "@webstudio-is/sdk-components-animation") {
-        features.set(
-          "Animation component",
-          findAwarenessByInstanceId(pages, instances, instance.id)
-        );
+        features.set("Animation component", {
+          awareness: findAwarenessByInstanceId(pages, instances, instance.id),
+        });
+      }
+    }
+
+    const badgeFeature = 'Remove "Built with Webstudio" badge';
```

Comment:
Remove?

Reply:
Missing?

Reply:
No, the feature is "No Badge" or Not having to add the badge